.github/workflows/mac-artifact-jfrog.yml

```diff
@@ -0,0 +1,123 @@
+name: Mac Artifact
+
+on:
+  repository_dispatch:
+    types: mac-build
+  push:
+    branches: [actionsHub, master, "bugfix-*"]
+  pull_request:
+    branches: [master]
+  workflow_call:
+    inputs:
+      submodule:
+        description: The directory of the submodule, if this workflow is being called on a submodule
+        required: false
+        type: string
+
+jobs:
```

Comment:
Can we do this with the reusable build workflow (and strip away much of the conditional (since it is for the tools 'meta' package) and the cache? Similar to https://github.com/aerospike/aql/pull/46/files#diff-004bcf41cd6b78932e3c9cfdcb3fcc79959f01eef1e685b8d8ad1b5da398dea7

Reply:
Will drop mac artifact from this pr

Reply:
dropped